src/lib/common/FileInput.svelte

```diff
@@ -19,8 +21,18 @@
 
 <!-- TODO Interactive elements inside a label are apparently invalid, but this works -->
 <label>
-  <input type="file" id={uniqueId} bind:this={fileInput} on:change={onChange} />
-  <button type="button" on:click={() => fileInput.click()}>{label}</button>
+  <input
+    type="file"
+    id={uniqueId}
+    bind:this={fileInput}
+    on:change={onChange}
+    disabled={$isAToolInUse}
```

Comment:
Hmm, this component is also used in `BrowseSchemes`, where this store is meaningless. Can we pass in a new `export let disabled = false` prop here, and from `EntireScheme`, set it to this store? I think changes to the store should propagate through properly